packages/blade/src/components/Accordion/index.ts

```diff
@@ -1,2 +1,4 @@
-export * from './Accordion';
+export { Accordion as Accordion_PRE_RELEASE } from './Accordion';
```

Comment:
once we do stable release people will have to manually remove this PRE_RELEASE? 😢

Reply:
Yup they'll have to change `Accordion_PRE_RELEASE` imports to `Accordion`. I don't forsee any API changes so should be the only change.